ndb/noxfile.py

```diff
@@ -68,6 +68,25 @@ def cover(session):
     session.run("coverage", "erase")
 
 
+@nox.session(py=DEFAULT_INTERPRETER)
+def lint(session):
+    """Run linters.
+    Returns a failure if the linters find linting errors or sufficiently
+    serious code quality issues.
+    """
+    session.install("flake8", "black")
+    session.run(
+        "black",
```

Comment:
Seems like this is duplicating a bit of the code in `def blacken()`?

Reply:
This checks that you blacked. Since CI doesn't apply the formatting changes.

Reply:
I realize, but I'm saying we could avoid repeating ourselves a bit by having a helper that takes `session` as an argument and an optional flag that turns on `--check`? E.g. so the file paths don't end up out of sync

Reply:
Now I understand. Yeah. we could certainly do that. This is being served as a template in the other libraries (as part of googleapis/synthtool) and would need to be updated there. This library has its own noxfile though. If you want to update it here I am happy to propagate that change to the others.

Reply:
> This library has its own noxfile though

Should we just get rid of that feature here?

> If you want to update it here I am happy to propagate that change to the others.

I think having `nox -s blacken` is useful, especially for cases when devs don't have editor support built in.